### DISCUSSION TITLE
How to verify setup is correct?

### DISCUSSION THREAD

**Discussion by GollyJer** [zachowj/hass-node-red#116] [Q&A] [answered]
Hi. I'm trying to get my Lutron switches set up through home-assistant into Node Red.

I believe I have everything set up correctly because the switch is detected and I'm able to configure it.

<img width=500 src=https://user-images.githubusercontent.com/689204/151740731-2e12abb1-6d63-4fb5-8ca1-fe10cb2b190f.png>

But, when I try and log the press nothing shows up.

<img width=400 src=https://user-images.githubusercontent.com/689204/151740885-ec2bb176-0404-4298-a317-1c1b94a6b567.png>

I also have hue-magic set up and I can see the button being pressed with that node. (the blue one in the screenshot)

I installed `hass-node-red`  via HACS.  It seems pretty fool proof but I'm proving to be a fool.

How can I verify I have things set up correcty?
Thanks.

**Comment by zachowj** [comment, marked as answer]
When the custom component is fully set up and working. The status text under the device node should say registered. I am guessing you didn't fully complete step 8 <https://github.com/zachowj/hass-node-red#installation>

**Reply by GollyJer** [reply]
Nailed it. Thanks. 👍

I created a PR with some clarity in the instructions. Not perfect but updated given what I know and the new configuration flow in HA 7.x.
https://github.com/zachowj/hass-node-red/pull/117
